libs/SalesforceSDKCore/Configuration/SalesforceSDKCore-Common-Static.xcconfig

```diff
@@ -10,5 +10,3 @@
 
 INSTALL_PATH = /
 SKIP_INSTALL = YES
-
-HEADER_SEARCH_PATHS = $(BUILT_PRODUCTS_DIR)/Headers $(TARGET_BUILD_DIR)/Headers $(inherited)
```

Comment:
@khawkins has been doing some work on cleaning up our targets and fixing everything in the workspace, so I'll defer to him on this.